.github/workflows/build.yml

```diff
@@ -206,7 +206,7 @@ jobs:
 
       - id: deploy
         name: Deploy
-        uses: aneoconsulting/ArmoniK.Action.Deploy/deploy@main
+        uses: aneoconsulting/ArmoniK.Action.Deploy/deploy@dd/fixUploadArtifact
```

Comment:
Revert back to main. The PR on Action.Deploy has been merged (https://github.com/aneoconsulting/Armonik.Action.Deploy/pull/7)

Reply:
@dbrasseur-aneo can you do it ?